tex/iii.1/_0086_0087_0088_0089_0090_0091c.tex

```diff
@@ -157,13 +161,12 @@ \section{Вплив обороту на норму зиску}
 оборотів змінного капіталу — 5. Отже, продукт кожного періоду
 обороту змінного капіталу буде:\[
 200c \text{(зношування)} \dplus{} 1000 c \dplus{} 1000 v \dplus{} 1000 m \deq{} 3200,\]
-
 а весь річний продукт при п’яти оборотах:
 
 \begin{center}
 $1000c\text{(зношування)} \dplus{} 5000 c \dplus{} 5000 v \dplus{} 5000 m \deq{} \num{16000}$,
 
-$К \deq{} \num{11000}, m \deq{} 5000, р' \deq{} \frac{5000}{\num{11000}} \deq{} 45\sfrac{5}{11}\%$.
+$К \deq{} \num{11000}, m \deq{} 5000, р' \deq{} \sfrac{5000}{\num{11000}} \deq{} 45\sfrac{5}{11}\%$.
```

Comment:
generally it's better not to use \sfrac in bloack equations (equations inside {center} or \[\]

Reply:
Okay, I have reverted them back